spec/services/proofing/aamva/request/verification_request_spec.rb

```diff
@@ -103,6 +103,12 @@
 
         expect(result.success?).to eq(true)
       end
+
+      it 'sends state id jurisdiction to AAMVA' do
+        applicant.state_id_data.state_id_jurisdiction = 'NY'
+
+        expect(subject.body).to match(/MessageDestinationId>NY/)
```

Comment:
I prefer to traverse the XML using a lib, but it's better than the status quo so 👍

Reply:
We have Nokogiri all over the app, we can use that here

examples:

https://github.com/18F/identity-idp/blob/ce7585e7e1f309b2ce870d6048c01fee20cc9c14/spec/views/layouts/application.html.erb_spec.rb#L75-L76

https://github.com/18F/identity-idp/blob/ce7585e7e1f309b2ce870d6048c01fee20cc9c14/spec/lib/telephony/otp_sender_spec.rb#L13

Reply:
I updated the spec to use nokogiri.  I'll merge this once the build passes.